builds/android/app/src/main/java/org/libsdl/app/SDLActivity.java

```diff
@@ -1213,10 +1221,10 @@ else if (event.getAction() == KeyEvent.ACTION_UP) {
     }
 
     // Touch events
-    /* EASYRPG: TOUCH DISABLED
+    /* EasyRPG modification: disable touch input
     @Override
     public boolean onTouch(View v, MotionEvent event) {
-        // Ref: http://developer.android.com/training/gestures/multi.html
+        /* Ref: http://developer.android.com/training/gestures/multi.html */
```

Comment:
Yeah sure, java does not seem to like this... :-1:
I just thought they had changed this in the SDL source and copied it over, but it was you when commenting out the whole stuff here. xD

Will quickly rebase this and then this PR is ready.